RecoEgamma/EgammaPhotonProducers/python/reducedEgamma_cfi.py

```diff
@@ -1,8 +1,8 @@
 import FWCore.ParameterSet.Config as cms
 
 reducedEgamma = cms.EDProducer("ReducedEGProducer",
-  keepPhotons = cms.string("pt > 14 && hadTowOverEm()<0.15"), #keep in output
-  slimRelinkPhotons = cms.string("pt > 14 && hadTowOverEm()<0.15"), #keep only slimmed SuperCluster plus seed cluster
+  keepPhotons = cms.string("(pt > 14 && hadTowOverEm()<0.15) || (pt>10 && pt<=14 && hadTowOverEm()<0.15 && chargedHadronIso()<10)"), #keep in output
```

Comment:
@cmkuo 
a much simpler logic that does the same thing would be 
hadTowOverEm()<0.15 && pt>10 && ( pt>14 || chargedHadronIso()<10)